### DISCUSSION TITLE
zotero6.0 插入模板时显示TypeError: Zotero.Knowledge4Zotero is undefined

### DISCUSSION THREAD

**Discussion by byhwsh** [windingwind/zotero-better-notes#795] [Q&A] [answered]
不会用better notes的模板，插入时总是显示错误。

Template [QuickBackLink] Error: TypeError: Zotero.Knowledge4Zotero is undefined

这种情况该如何处理呢？此外，zotcard的笔记模板可以正常使用。
谢谢各位大佬！

**Comment by windingwind** [comment, marked as answer]
1. 模板问题向模板提供者反馈
2. 插件使用问题请具体描述
3. 其他插件的问题此处不做回答
4. 看起来你在使用一个过于陈旧的模板
